ui/core/src/model/query.ts

```diff
@@ -12,16 +12,35 @@
 // limitations under the License.
 
 import { Definition, UnknownSpec } from './definitions';
+import { TimeSeriesData } from './time-series-data';
 
 interface QuerySpec<PluginSpec> {
   plugin: Definition<PluginSpec>;
 }
 /**
  * A generic query definition interface that can be extended to support more than just TimeSeriesQuery
  */
-export interface QueryDefinition<Kind = QueryType, PluginSpec = UnknownSpec> {
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export interface QueryDefinition<Kind = any, PluginSpec = UnknownSpec> {
```

Comment:
Would an `unknown` kind work better here? If it's important to use `any` could you put a comment above the `eslint-disable` explaining why for future people working in the code.

Reply:
I had to use `any` here because otherwise I was getting typescript errors complaining that unknown cannot be converted to TimeSeriesQueryDefinition